tools/lsp/language/signature_help.rs

```diff
@@ -0,0 +1,334 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+use crate::common::DocumentCache;
+use i_slint_compiler::expression_tree::Expression;
+use i_slint_compiler::langtype::{ElementType, Type};
+use i_slint_compiler::lookup::{LookupObject as _, LookupResult};
+use i_slint_compiler::namedreference::NamedReference;
+use i_slint_compiler::parser::{syntax_nodes, SyntaxKind, SyntaxNode, SyntaxToken};
+use lsp_types::{ParameterInformation, ParameterLabel, SignatureHelp, SignatureInformation};
+
+pub(crate) fn get_signature_help(
```

Comment:
Isen't `pub` enough here? This is a binary anyway.

Reply:
Yes, pub would work.  I think it was auto-completed by rust-analyzer.